modules/search/pages/customize-index.adoc

```diff
@@ -13,11 +13,13 @@ You can add the following components and configure the following options for a S
 
 |[[type-identifiers]]Type Identifier a|
 
-Set a type identifier to change how the Search index finds a document's type.
+The Type Identifier offers three optional methods for filtering documents from your collections to be included in a Search index:
 
-The type identifier tells the Search Service the document field or location in the document's ID value to use as the document's type. 
+* JSON Type Field: Selects only documents that contain a specific field with a particular string value.
```

Comment:
Can we please get some clarity from say, Abhi, on what exactly the type identifier does? The equivalent parameters on the JSON, you didn't edit like this, and yet now it's apparently wrong here? I want to know what exactly this does if I've apparently gotten it wrong.